### PR TITLE
USEID-915: Enable export per record

### PR DESCRIPTION
**Pull request by pgurusinga**

### CODE REVIEW

File: query_and_submit.py [hunk 1]
```diff
@@ -21,19 +21,49 @@
...
+      url=config["endpoint"],
+      json={
+        "startInterval": interval_start,
+        "endInterval": interval_end,
+        **metrics,
+      },
+      headers={
+        "xc-token": metrics_token,
+      }
+    )
```
Comment (pgurusinga) [inline query_and_submit.py]: If this post is not inside the else, it will post empty record

**Comment by pgurusinga** [comment]
there are many critical and high-severity security vulnerabilities, in the base branch and the PR branch.

https://github.com/digitalservicebund/kpi-exporter-postgres/pull/1/checks?check_run_id=12023783533 

https://hub.docker.com/layers/library/python/3.10.5-alpine/images/sha256-70ee0541a51dfb65c8b014de3e891b5ed3ffeebde23db79169e7ba4490af6ed9?context=explore

I guess, with my limited Python knowledge, we need to upgrade, to `3.10.10`

File: query_and_submit.py [hunk 2]
```diff
@@ -21,19 +21,49 @@
...
+    for result in db_result:
+      category, content, created_at = result.split("|")
+
+      content_metric["category"] = category
+      content_metric["content"] = content
+      content_metric["date"] = created_at
+
+      json_output = {
+        "createdAt": interval_start,
+        "endInterval": interval_end,
```
Comment (JulianRoesner) [inline query_and_submit.py]: I am wondering whether we still need those fields? Are those columns used in the table?

File: query_and_submit.py (hunk 1 above)
Comment (JulianRoesner) [reply query_and_submit.py]: Ah, makes sense. However, if this is in the else it won't work to collect all other metrics in one entry as before.

There are two ways I could see how we can fix that: 
- Either remove that part completely and have two different jobs (one for content and one for count) using two different images.
- Or we use the bulk post method and only post once at the end.

We can also talk about it.

**Review by JulianRoesner — COMMENTED** [review]
Looks good to me but I think that we still need to adapt the else case.

File: query_and_submit.py [hunk 3]
```diff
@@ -18,22 +18,52 @@
...
+  json=metrics,
   headers={
     "xc-token": metrics_token,
   }
 )
+
+if response.ok:
+  print("export done")
+else:
+  print("export failed")
```
Comment (JulianRoesner) [inline query_and_submit.py:69]: Like that 👍

**Review by JulianRoesner — APPROVED** [review]
LGTM 🚀

**Pull request merged**